build_system.sh

```diff
@@ -46,45 +48,50 @@ echo "Building image"
 export DOCKER_CLI_EXPERIMENTAL=enabled
 docker build -f Dockerfile.agnos -t agnos-builder $DIR
 
+# Setup mount container
+MOUNT_CONTAINER_ID=$(docker run -d --privileged --volume $BUILD_DIR:$BUILD_DIR ubuntu:latest sleep infinity)
```

Comment:
what's this needed for?

Reply:
the namespace runners have some permission issues, even when using sudo - can't use (sudo) mount/unmount - so you mount them in a container (also works on mac with this change)